Sources/UI/Crash Logging/CrashLoggingView.swift

```diff
@@ -97,14 +97,11 @@ extension CrashLoggingView {
         let error = SentryTestError(title: "Test Error")
 
         do {
-            try crashLogging.logErrorImmediately(error) { result in
-                switch result {
-                    case .success:
-                        sendErrorAndWaitStatus = .success
-                    case .failure(let err):
-                        sendingError = err
-                        sendErrorAndWaitStatus = .error
-                }
+            try crashLogging.logErrorImmediately(
+                error,
+                userInfo: ["custom-userInfo-key": "custom-userInfo-value"]
+            ) {
+                sendErrorAndWaitStatus = .success
```

Comment:
If we decide to adopt `flush`, it might be appropriate to update this `enum` to remove the error-tracking state, since there's no longer any error information with `flush`.